python/lsst/ts/ess/common/device/mock_csat3b_formatter.py

```diff
@@ -0,0 +1,72 @@
+# This file is part of ts_ess_common.
+#
+# Developed for the Vera C. Rubin Observatory Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+__all__ = ["MockCsat3bFormatter"]
+
+import random
+
+from ..sensor import compute_signature
+from .mock_formatter import (
+    MockFormatter,
+    MockTemperatureConfig,
+    MockWindSpeedConfig,
+)
+
+
+def format_csat3b_temperature(index: int, missed_channels: int) -> str:
+    if index < missed_channels:
+        return ""
+    else:
+        value = random.uniform(MockTemperatureConfig.min, MockTemperatureConfig.max)
+        return f"{value:6.5f}"
+
+
+def format_csat3b_wind_speed(index: int, missed_channels: int) -> str:
+    if index < missed_channels:
+        return ""
+    else:
+        value = random.uniform(MockWindSpeedConfig.min, MockWindSpeedConfig.max)
+        return f"{value:6.5f}"
+
+
+class MockCsat3bFormatter(MockFormatter):
+    def __init__(self) -> None:
+        self.count = 0
+
+    def format_output(
+        self,
+        num_channels: int = 0,
+        disconnected_channel: int = 0,
+        missed_channels: int = 0,
+    ) -> list[str]:
+        x = format_csat3b_wind_speed(index=0, missed_channels=missed_channels)
+        y = format_csat3b_wind_speed(index=1, missed_channels=missed_channels)
+        z = format_csat3b_wind_speed(index=2, missed_channels=missed_channels)
+        t = format_csat3b_temperature(index=3, missed_channels=missed_channels)
+        d = 0
+        self.count = self.count + 1
+        if self.count > 63:
```

Comment:
Please consider a named constant instead of 63, e.g. _MAX_COUNT

Also, you might consider a one-liner:
```
self.count = (self.count + 1) % (_MAX_COUNT + 1)
```

Reply:
Done